third_party/types/common-types.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as http from 'http';
```

Comment:
Do you intend to commit this third_party/types/common-types.ts file? I'd expect this to be a Node.js dependency?

Reply:
Yes -- right now @google-cloud/commons does not have type definitions. So, I'm borrowing the type definitions defined by cloud-debug-nodejs.